src/rules/rule-non-nested-empty-line-before/README.md

```diff
@@ -132,3 +132,51 @@ a
 b
 {}
 ```
+
+## Optional options
+
+### `except: ["after-single-line-comment"]`
+
+Reverse the primary option if the rule is after a single line comment.
+
+For example, with `"always"`:
+
+The following patterns are considered warnings:
```

Comment:
Lets simplify these even further to just:

---

For example, with `"always"`:

The following patterns are considered warnings:

```css
/* comment */

a {}
```

The following patterns are *not* considered warnings:

```css
/* comment */
a {}
```

---